coffee_lib/src/types/mod.rs

```diff
@@ -1,58 +1,90 @@
 //! Coffee Model Definition
-use serde::{Deserialize, Serialize};
 
-use crate::plugin::Plugin;
+// Definition of the request types.
+pub mod request {
+    use paperclip::actix::Apiv2Schema;
```

Comment:
```suggestion
    #[cfg(open-api)]
    use paperclip::actix::Apiv2Schema;
```

this is not present aways